src/content/linkto.rs

```diff
@@ -0,0 +1,250 @@
+use ssri::{Algorithm, Integrity, IntegrityOpts};
+use std::fs::DirBuilder;
+use std::fs::File;
+use std::path::{Path, PathBuf};
+use std::pin::Pin;
+use std::task::{Context, Poll};
+
+use crate::async_lib::AsyncRead;
+use crate::content::path;
+use crate::errors::{IoErrorExt, Result};
+
+#[cfg(not(any(unix, windows)))]
```

Comment:
I thought the correct way to do this was `target_family = "unix"/"windows"`. I'm not sure this conditional does anything. Does it?? I guess it must've compiled for you, so...

Reply:
https://doc.rust-lang.org/reference/conditional-compilation.html#unix-and-windows

> `unix` is set if `target_family = "unix"` is set and `windows` is set if `target_family = "windows"` is set.

Happy to use the more explicit `target_family` if you prefer

Reply:
nope, I'm happy with this :) Just making sure I was getting it right. I've been having a lot of Adventures™️ with cfg recently :P